src/hathor.c

```
@@ -53,6 +62,10 @@ void address_from_pubkey_hash(const uint8_t *public_key_hash, uint8_t *out) {
 
 void address_from_pubkey(cx_ecfp_public_key_t *public_key, uint8_t *out) {
     uint8_t buffer[PUBKEY_HASH_LEN] = {0};
+
+    if ((public_key == NULL) || (out == NULL)) {
+        return;
```

Are failing silently here?